spec/controllers/administrateurs/procedures_controller_spec.rb

```diff
@@ -761,4 +761,17 @@ def update_monavis
       it { expect(procedure.allow_expert_review).to be_truthy }
     end
   end
+
+  describe 'PUT #restore' do
+    let!(:procedure) { create :procedure, :discarded, administrateur: admin }
```

Comment:
`let!` est un code-smell (ça rend difficile d'organiser l'ordre dans lequel sont exécutés les initializers). Ici on peut utiliser juste `let` sans souci.

```suggestion
    let(:procedure) { create :procedure, :discarded, administrateur: admin }
```